spec/index.bs

```diff
@@ -1759,7 +1759,7 @@ applications should generally request the "worst" limits that work for their con
         {{GPUComputePassEncoder/dispatchWorkgroups(workgroupCountX, workgroupCountY, workgroupCountZ)}}.
 </table>
 
-#### <dfn interface>GPUSupportedLimits</dfn> #### {#gpu-supportedlimits}
+<h5 id=gpusupportedlimits data-dfn-type=interface>`GPUSupportedLimits`</h5>
```

Comment:
Here and elsewhere throughout the doc:

Markdown syntax typically defines `#` as `h1`, `##` as `h2`, and so on, but you use `<h(# count+1)>` throughout. I assume that you've checked this against the rendered doc and that Bikeshed simply does it a bit differently than standard Markdown, but I wanted to sanity check it really quick. :)

Reply:
Yes, if I do it the markdown way, Bikeshed errors that the heading level jumped (e.g. from 3 to 5). Bikeshed only emits one h1 (the title) and the main headings are all h2.